### DISCUSSION TITLE
option value not recorded

### DISCUSSION THREAD

**Discussion by IanMuller** [YetiForceCompany/YetiForceCRM#16532] [How to]
hi, trying to put picklist values from API connected form which are sadly not recorded. Is there any trick?
Using standard html syntax's such as:

` <select multiple="multiple" name="Source_language">
    <option value="Arabic">Arabic</option>
    <option value="Bulgarian">Bulgarian</option>
    <option value="Burmese">Burmese</option>
  </select>`

All other fields are correctly recorded

**Comment by mariuszkrzaczkowski** [comment]
give more details

**Reply by IanMuller** [reply]
The thing is the multipicklist is a different table in this case: vtiger_multipicklist.

![image](https://user-images.githubusercontent.com/34973571/192925052-763361ab-da8a-4579-a7bd-7fa68ad9d8be.png)

**Reply by IanMuller** [reply]
![image](https://user-images.githubusercontent.com/34973571/192925975-06b3170d-6e35-4d44-ad5a-60218a485f60.png)

**Reply by mariuszkrzaczkowski** [reply]
use the `/webservice/WebserviceStandard/{moduleName}/Fields` method to get the multipicklist values
https://doc.yetiforce.com/api/#/BaseModule/2a1d7369646775c9090f7f3a3c8c8322

**Comment by s2b-git** [comment]
Me also not sure how to put picklist value using API. Not a programmer but trying :-) 
Any guide?

Best
